### PR TITLE
Added unwind hashstate

### PR DESCRIPTION
**Pull request by Giulio2002**

### CODE REVIEW

File: libmdbx/mdbx.h [hunk 1]
```diff
@@ -738,7 +738,7 @@ enum MDBX_constants {
 /** \brief The name of the lock file in the environment */
 #define MDBX_LOCKNAME "/mdbx.lck"
 /** \brief The name of the data file in the environment */
-#define MDBX_DATANAME "/mdbx.dat"
+#define MDBX_DATANAME "mdbx.dat"
```
Comment (AndreaLanfranchi) [inline libmdbx/mdbx.h:741]: Please revert this.
We can't change MDBX defs as they would get overwritten when MDBX code is updated

**Review by AndreaLanfranchi — CHANGES_REQUESTED** [review]

File: cmd/dbtool.cpp [hunk 2]
```diff
@@ -161,6 +162,11 @@ struct copy_options_t {
     size_t filesize{0};                  // Size of target file if exists
 };
 
+struct stage_set_options_t {
```
Comment (yperbasis) [inline cmd/dbtool.cpp:165]: For consistency with our style, please name it StageSetOptions

**Comment by Giulio2002** [comment]
Ready for review

**Review by yperbasis — APPROVED** [review]

**Comment by yperbasis** [comment]
LGTM. @AndreaLanfranchi are you happy with the PR?

**Review by AndreaLanfranchi — CHANGES_REQUESTED** [review]

**Comment by AndreaLanfranchi** [comment]
> LGTM. @AndreaLanfranchi are you happy with the PR?

Generally yes ... question for @Giulio2002 have you tried some unwind methods on a synced db and eventually reuse that db for normal syncing on Erigon ? Is the db still consistent ?

**Review by yperbasis — APPROVED** [review]

**Pull request merged**